index.ts

```diff
@@ -97,6 +103,50 @@ interface LoginOptions {
   localhost: boolean;
 }
 
+interface CreateOptions {
+  env: string;
```

Comment:
We can't just add env options to every command. They need to be in `.clasp.json`.
https://github.com/google/clasp/issues/12#issuecomment-370672417

Reply:
I think that `env` option is need to specify envId in `.clasp.json`. If no `env` option, How do it specify `envId` from CLI ?

Reply:
I don't support a --env flag in each CLI command. Follow https://github.com/google/clasp/issues/12#issuecomment-370672417 carefully. It's very complex and I don't know if I covered all the cases. We should probably discuss more before implementing.

Reply:
I understood your idea.
I have a consideration for use-case of multiple google accounts and multiple `scriptId`s at same time. Please see https://github.com/google/clasp/issues/12#issuecomment-371168118